app/search/controllers.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from flask import Blueprint, request, jsonify
+from flask import current_app as app
+from app.utils import handle_error
+from app.search.models import DataPackageQuery
+
+search_blueprint = Blueprint('search', __name__, url_prefix='/api/search')
+
+
+@search_blueprint.route("/package", methods=["GET"])
+def search_packages():
+    """
+        DPR metadata put operation.
+        This API is responsible for searching data package
+        ---
+        tags:
+            - search
+        parameters:
+            - in: query
+              name: q
+              type: string
+              required: true
+              description: search query string e.g. q=query publisher=pub
+        responses:
+            500:
+                description: Internal Server Error
+            200:
+                description: Success Message
+                schema:
+                    id: search_package_success
+                    properties:
+                        status:
+                            type: string
+                            description: Status of the operation
+                            default: SUCCESS
+                        data:
+                            type: list
+                            properties:
+                                type: object
+                                properties:
+                                    name:
```

Comment:
Why are we not returning the whole datapackage object. I suggest that we should have here is an array with each item being what you'd get from a call to package GET API.

In addition, you should have a total attribute that tells you the total number of results.

BTW this response looks different from what we specified in the epic for this: https://github.com/frictionlessdata/dpr-api/issues/200 (though that probably needs some refinement e.g. `results` rather than `packages` for list of results etc.